optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -42,13 +42,13 @@
 _LGBM_PARAMS_KEY = "lightgbm_tuner:lgbm_params"
 
 # EPS is used to ensure that a sampled parameter value is in pre-defined value range.
-EPS = 1e-12
+_EPS = 1e-12
```

Comment:
Do we need to make this type of variables private? If so, we need to change other variables (e.g. `EPS` in `optuna.samplers.tpe.sampler.py` for consistency.

Reply:
I agree that it's open for discussion. Maybe contradictory to how we implicitly propagate private-ness between packages and modules. How about leaving this PR as is since it covers not just `optuna.integration.lightgbm_tuner` but the entire code base, and continue this discussion elsewhere?

Reply:
Thank you for your comments. 

> How about leaving this PR as is since it covers not just optuna.integration.lightgbm_tuner but the entire code base, and continue this discussion elsewhere?

I agree with you. We can work on this issue in a new PR or issue.